docs/references/expo/expo-web-support.mdx

```diff
@@ -0,0 +1,106 @@
+
+### Build sign in and sign up pages with prebuilt components on web
+
+This guide shows you how to use the [`<SignIn />`](/docs/components/authentication/sign-in) and [`<SignUp />`](/docs/components/authentication/sign-up) prebuilt components in order to build custom sign-in and sign-up pages for your Expo web app.
+
+For this guide we will leverage [Expo Router](https://docs.expo.dev/router/introduction/) and the [platform-specific extensions](https://docs.expo.dev/router/create-pages/#platform-specific-extensions) to build the sign-in and sign-up pages specifically for the web platform.
+
+<Steps>
+  ### Build a sign-up page
+
+  Create a new file `sign-up.web.tsx`, which will be used to render the sign-up page. In the file, import the [`<SignUp />`](/docs/components/authentication/sign-up) component from `@clerk/clerk-expo/web` and render it.
+
+  ```tsx filename="/app/sign-up.web.tsx"
+  import { SignUp } from '@clerk/clerk-expo/web'
+
+  export default function Page() {
+    return <SignUp />
+  }
+  ```
+
+  ### Build a sign-in page
+
+  Create a new file `sign-in.web.tsx`, which will be used to render the sign-in page. In the file, import the [`<SignIn />`](/docs/components/authentication/sign-in) component from `@clerk/clerk-expo/web` and render it.
+
+  ```tsx filename="/app/sign-in.web.tsx"
+  import { SignIn } from '@clerk/clerk-expo/web'
+
+  export default function Page() {
+    return <SignIn />
+  }
+  ```
+
+  ### Visit your new pages
+
+  Run your project with the following terminal command from the root directory of your project:
+
+  <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+    ```bash filename="terminal"
+    npm run web
+    ```
+
+    ```bash filename="terminal"
+    yarn web
+    ```
+
+    ```bash filename="terminal"
+    pnpm web
+    ```
+  </CodeBlockTabs>
+
+  Visit your new custom pages locally at [localhost:8081/sign-in](http://localhost:8081/sign-in) and [localhost:8081/sign-up](http://localhost:8081/sign-up).
+</Steps>
+
+## More resources
+
+You can also learn more about Clerk components and how to customize them and also how build custom authentication flows, and how to use Clerk's client side helpers. The following guides will help you get started.
+
+<div className="container mx-auto my-4">
+  <div className="grid grid-cols-1 gap-6 md:grid-cols-2">
+    <Cards
+      title="Prebuilt Components"
+      description="Learn more about Clerk's suite of components that let you quickly add authentication to your app."
+      link="/docs/components/overview"
+      cta="Learn More"
+    />
+
+    <Cards
+      title="Customization & Localization"
+      description="Learn how to customize and localize Clerk components."
+      link="/docs/components/customization/overview"
+      cta="Learn More"
+    />
+
+    <Cards
+      title="Custom flows"
+      description="Learn how to build custom authentication flows."
+      link="/docs/custom-flows/overview"
+      cta="Learn More"
+    />
```

Comment:
I wouldn't link people to custom flows - we only want people to use custom flows for advanced situations. We don't want to advertise them

```suggestion
```

Reply:
Yeah but for Expo it's the only way right now in order to build universal apps